Arena/src/Arena/Views/MainWindow.xaml

```diff
@@ -19,6 +19,9 @@
             </ResourceDictionary.MergedDictionaries>
             <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
             <converters:InverseBooleanConverter x:Key="InverseBooleanConverter"/>
+            <converters:NormalSpeedToNegativeBoolConverter x:Key="NormalSpeedToNegativeBoolConverter"/>
```

Comment:
Instead of coming up with such an unusual converter let's create IsEnabled property for those commands and bind it properly